_co-op-companies/A-bold.md

```diff
@@ -0,0 +1,17 @@
+---
+title: "Bold Commerce"
+identification: "bold"
+description: "Bold is an eCommerce company."
+link: "https://boldcommerce.com/"
+image: "assets/img/logos/bold-icon.png"
+width: "100px"
+members:
+  - name: "Elizabeth Kenyon"

```

Comment:
We'd like to just display first names.

Reply:
Currently, the name is just for identification in the markdown, and only the "summary" and "statement" Strings are pulled onto the page.

Reply:
Okay great! :+1: